plugins/modules/source_control/gitlab/gitlab_group_members.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Zainab Alsaffar <Zainab.Alsaffar@mail.rit.edu>
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
```

Comment:
Please remove this. It is no longer needed.

Reply:
Resolved! 

Thanks for reviewing the code and providing feedback. I was following the instruction at: [https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html](Module format and documentation)

Would you please direct me to the updated version so I can use it for any future contribution ?